### PR TITLE
[LOOP-3842/3843] Capture trend rate and out-of-range condition in glucose sample

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-3842
- https://tidepool.atlassian.net/browse/LOOP-3843
- https://tidepool.atlassian.net/browse/LOOP-970
- https://tidepool.atlassian.net/browse/LOOP-1257
- Add glucose trend rate and glucose condition

### CODE REVIEW

**Comment by darinkrauss** [comment]
LW PR for build: https://github.com/tidepool-org/LoopWorkspace/pull/648

**Review by rickpasetto — APPROVED** [review]
LGTM

File: ShareClient/ShareGlucose+GlucoseKit.swift [hunk 1]
```diff
@@ -41,6 +45,12 @@ extension ShareGlucose: GlucoseDisplayable {
     }
 }
 
+extension ShareGlucose {
+    public var condition: GlucoseCondition? {
```
Comment (ps2) [inline ShareClient/ShareGlucose+GlucoseKit.swift:53]: Wouldn't we return high/low for 401/39 here?

Comment (darinkrauss) [reply ShareClient/ShareGlucose+GlucoseKit.swift:53]: Well, we can, but I wasn't sure if the 39/401 limit were the same for all versions of Dexcom. I figured that DIY folks could make that determination. However, if you are telling me that all versions of Dexcom use 39 and 401 as LOW and HIGH indicators and we can do the same thing as in Tidepool Loop then I will update this code accordingly.

Comment (darinkrauss) [reply ShareClient/ShareGlucose+GlucoseKit.swift:53]: @ps2 What would you like me to do here?

Comment (darinkrauss) [reply ShareClient/ShareGlucose+GlucoseKit.swift:53]: Updated to use 40/400 limits for both `HKQuantity` and `GlucoseCondition`.

**Pull request merged**